### ISSUE TITLE
Some aspects are not obvious

### ISSUE THREAD

**Issue by devkennyy** [devkennyy/rungeon#35]
I've told a few people to checkout out the site and they don't know to click on the dungeon doors, they don't know to squash the bug icon. Should we make it more obvious or leave it as is since it's a game.

**Comment by Andreu-G** [comment]
To be honest, it felt pretty clear to me when I first checked it. There isn't much on the screen to get lost anyway.
If anything, we could make the icons change color when hovering over them.

**Comment by devkennyy** [comment]
You're right, the people I sent this too aren't very tech literate anyways. We could still change the opacity on hover, or stop the blink animation for the doors?

**Comment by Andreu-G** [comment]
For the index animation, we could maybe add 2 [triangle arrows ](https://fontawesome.com/icons/circle-caret-right?s=solid) if you don't want to modify the color/animation.

**Comment by devkennyy** [comment]
I'm not sure what you mean by 2 triangle arrows? Side by side?

**Comment by Andreu-G** [comment]
Yes. If you click on the link, you'll see the fontawesome icons I was talking about. I meant having one [right-arrow](https://fontawesome.com/icons/circle-caret-right?s=solid) on the left side, and one [left-arrow ](https://fontawesome.com/icons/circle-caret-left?s=solid) on the right side of the door icon, to further specify that they have to do something with the door (in this case, to click on it).

**Comment by devkennyy** [comment]
Oh, I understand! If it looks nice, I'm down.

**Comment by Shabbyconnor** [comment]
Some sort of mouseover highlight or color change on the starting door might be helpful. When I first saw it I thought it was a loading screen rather than something I should click.

**Comment by devkennyy** [comment]
I paused the animation, reset the opacity and upped the font-size a little. If this isn't enough we can proceed with the arrows but I think it will suffice for now.

**Comment by Andreu-G** [comment]
> Some sort of mouseover highlight or color change on the starting door might be helpful. When I first saw it I thought it was a loading screen rather than something I should click.

Thought the same, although that won't work for mobile users since there's no mouseover.

**Comment by GruelingPine185** [comment]
I agree with what @Andreu-G said, but if I were to propose a solution, I'd have some sort of counter that counts down (from 10 for example) in the background and triggers either arrows that point to what the user has to click/press to proceed or have a small popup (something like a tooltip) that tells the user what to click/press. It's simple, but for people who don't like tutorials/guided stuff like me, it's a both distracting and annoying. Especially for returning users/people who are already familiar to the site.

**Comment by GruelingPine185** [comment]
It's honestly up to @devkennyy. I'd go with every stage

**Comment by Shabbyconnor** [comment]
Maybe we could add a hint button instead of a timer?

**Comment by devkennyy** [comment]
> It's simple, but for people who don't like tutorials/guided stuff like me, it's a both distracting and annoying. 

I agree, tutorials are tedious, unnecessary, and wouldn't go with the minimal theme at all. 

> I'd have some sort of counter that counts down (from 10 for example) in the background and triggers either arrows that point to what the user has to click/press to proceed or have a small popup (something like a tooltip) that tells the user what to click/press.

My initial vision for the project was that users would share the answers and anyone stuck would Google them but I don't see that happening anytime soon. The game is more puzzle sided than text adventure so I don't see the point in giving away the answers. Thoughts?

**Comment by GruelingPine185** [comment]
I agree. Rungeon is going to need to be a much larger game before anything like an answer forum should be created.

An observation:
When I play the game, it feels like something to be speed ran rather than enjoyed. Being a text adventure game, I do believe that it should utilize randomized prompts or maybe an AI. In doing so, it slows the end user down while also adding new taste to keep them engaged.
